fx-src/jsm_prefix.js

```diff
@@ -30,4 +30,4 @@
 // more details.
 const global = this;
 
-this.EXPORTED_SYMBOLS = ["Kinto"];
+var EXPORTED_SYMBOLS = ["Kinto"];
```

Comment:
What's this change about?

Reply:
https://bugzilla.mozilla.org/show_bug.cgi?id=1440284 :)